### PR TITLE
Add persistent comments using Datastore

### PR DESCRIPTION
**Pull request by maceonthompson**
Again, due to the way the walkthrough is setup, this pull request is predicated on the previous #12

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -14,30 +14,57 @@
...
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
-import java.util.Arrays;
+
 import com.google.gson.Gson;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Serlvet that enables commenting on the website. */
```
Comment (kristen-law) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:41]: Typo: Serlvet -> Servlet

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -14,30 +14,57 @@
...
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<Comment> comments = new ArrayList<Comment>();
+  // private List<Comment> comments = new ArrayList<Comment>();
```
Comment (kristen-law) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:45]: Please remove this if you're not using it.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -14,30 +14,57 @@
...
+
 import com.google.sps.data.Comment;
+
+//JavaIO imports
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+//Servlet imports
```
Comment (kristen-law) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:33]: You don't necessarily need to change this, but I'd say in general you don't need to label/comment on your imports if it's pretty clear from the package name what the import is for.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (maceonthompson) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Resolved!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (maceonthompson) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Resolved!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (maceonthompson) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:33]: I normally wouldn't label imports, but the cloud shell doesn't auto import (or auto-complete most times at all), so it was for me to keep track of imports as I did them. In the future would it be better practice to remove those comments before I commit?

Comment (kristen-law) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:33]: For this project I guess it's ok to leave them in if it helps you keep track of them. It definitely varies per project but in general I think it's better to remove unnecessary/redundant comments.

**Pull request merged**